1.20/s2i/bin/assemble

```diff
@@ -20,7 +20,6 @@ if [ -d ./nginx-cfg ]; then
     cp -av ./nginx-cfg/*.conf "${NGINX_CONFIGURATION_PATH}"
     rm -rf ./nginx-cfg
   fi
-  chmod -Rf g+rw ${NGINX_CONFIGURATION_PATH}
```

Comment:
Are we sure we can remove this? This is likely in the assemble script since we know the new config files added through s2i will have wrong permissions.

Reply:
It works because the assembly script in our tests is run by the 1001:0 user, and thus the files copied should be owned by 1001:0 by default. Using `cp -a` preserves permissions on copied files, which already have rw for group. 

So IMO these chmods in assembly scripts do not have any meaning (please correct me, if I understand this incorrectly) and can confuse users.

Reply:
I do not see much need for it either, but I might be missing something. I see the chmod calls were added by @hhorak in #41 so maybe he would remember the reason for adding it?
Considering that in the best case the calls are functionally-wise a no-op (since we expect all the permissions are already set), we might as well keep it in in as it would not make any difference.

Reply:
I'll probably not bring too much light on this -- my #41 was built on top of #32, so I mostly took over what was prepared by @kosciCZ back then. I'm wondering how much legal it is to run the assemble script by root -- I can imagine a user wants not only run the assemble but also install some RPMs before -- in that case few more chown calls might do the difference. (it's just and idea so far, but as @pkubatrh mentioned, maybe ok to keep it there if it shouldn't do anything in most common cases)

Reply:
I would say, that the assembly script can be run as root, when using rootfull container, with unspecified UID - then the default is AFAIK 0 (root).
However, it would not have to be a problem as the copied configuration files and directories have their permissions changed before, with `fix-permissions` here: https://github.com/sclorg/nginx-container/blob/a4db1d75aaac92c590ecebdc10b8c52d869444c3/1.20/s2i/bin/assemble#L9
That basically does g+rw on $1 (and that is what I have removed from the assembly script).

Anyway, I understand your point of view and if you think, that it would be safer to let those commands in assembly script I would agree, as I can not see how they could be harmful in any way.

Reply:
I have removed the changes in the assembly script.